packages/block-editor/src/components/off-canvas-editor/block.js

```diff
@@ -237,6 +236,10 @@ function ListViewBlock( {
 		? selectedClientIds
 		: [ clientId ];
 
+	const BlockSettingsDropdownComponent = LeafMoreMenu
```

Comment:
I think we should probably change the name of this variable as it might lead people to expect that it this is `BlockSettingsDropdown` but in some cases it may not be.